src/styles/index.scss

```diff
@@ -1,3 +1,150 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+:root {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+$color__text--main: #060b35;
+$color__text--secondary: #616070;
+$color__text--accent: #00acdc;
+$color__background--accent: #00acdc;
+
+.card {
```

Comment:
Please place styles for 'card' block into a separate file. Same for 'stars' block.